.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: false
+language: node_js
+node_js:
+  - "8"
+  - "10"
+  - "node" # latest stable Node.js release
+
+cache:
```

Comment:
Can simplify to [`cache: npm`](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#caching-with-npm).